src/jdk.jartool/share/classes/sun/security/tools/jarsigner/Main.java

```diff
@@ -30,6 +30,7 @@
 import java.net.URLClassLoader;
 import java.security.cert.CertPathValidatorException;
 import java.security.cert.PKIXBuilderParameters;
+import java.security.spec.PSSParameterSpec;
```

Comment:
I don't think you need this import, as this class does not seem to be referenced anywhere.

Reply:
Removed. Thanks for the review.